src/items/item.js

```diff
@@ -96,17 +97,17 @@ export class Item extends ClickableObject {
     }
 
     async buy() {
```

Comment:
does this have to be async? not sure if you're using that correctly. most of the time it's used for IO like getting stuff from the disk or api calls

Reply:
Need to be async to use await in line 104 `let boughtItem = await getItemClass(this.constructor.name, dashboard);`